src/custom.scss

```diff
@@ -3,12 +3,42 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
```

Comment:
might be good to have NER fonts in here?

Reply:
I just copied this from the index.css file, what are the NER fonts?